src/backends/torch/torchlib.h

```diff
@@ -53,6 +53,11 @@ namespace dd
     TorchModule();
 
     c10::IValue forward(std::vector<c10::IValue> source);
+    c10::IValue extract(std::vector<c10::IValue> source,
```

Comment:
Are the c10 extracts public ? or the class public functions may only manipulate `torch::Tensor` ?

Reply:
for now, extract can return only one tensor, but I have a ticket/card to make everything be able to return any IValue (ie a list of tensors if the layer do so [exactly IValue is a variant ;) ].  so this is kind of a spoiler :D